drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -153,6 +171,7 @@ function DragNDropTemplates(url_name) {
                         'dropzone': 'move',
                         'aria-dropeffect': 'move',
                         'data-zone': zone.title,
+                        'data-zone_align': zone.align,
```

Comment:
@pomegranited Nit: `data-zone_align` ⟶ `data-zone-align` (I think we generally stick to dashes for HTML attributes).

Also, I noticed that this leads to `data-zone-align` not having a value if "Alignment" is set to "none" for a zone:

![data-zone-align-non-aligned-zone](https://cloud.githubusercontent.com/assets/961441/12783901/484214d0-ca84-11e5-92a3-25ec374db5b1.png)

I think it would make sense to set the value of the attribute to `none` instead, to be consistent with the remaining alignment options.